cmd/relayproxy/config/config.go

```diff
@@ -175,6 +175,14 @@ type Config struct {
 	// Default: false
 	EnablePollingJitter bool `mapstructure:"enablePollingJitter" koanf:"enablepollingjitter"`
 
+	// DisableNotificationOnInit (optional) set to true if you do not want to
+	// send a notification when the flags are loaded.
+	// This is useful if you do not want a Slack/Webhook notification saying that
+	// the flags have been added every time you start the application.
+	// Default is set to false for backward compatibility.
+	// Default: false
+	DisableNotificationOnInit bool `mapstructure:"disableNotificationOnInit" koanf:"disablenotificationoninit"`
```

Comment:
While I understand why you've put `notification` here, but I think it will be more explicit to use the name `notifier` which is what we are using everywhere.


```suggestion
	// DisableNotifierOnInit (optional) set to true if you do not want to
	// send a notification when the flags are loaded.
	// This is useful if you do not want a Slack/Webhook notification saying that
	// the flags have been added every time you start the application.
	// Default is set to false for backward compatibility.
	// Default: false
	DisableNotifierOnInit bool `mapstructure:"disableNotificationOnInit" koanf:"disablenotificationoninit"`
```

Reply:
I think it makes sense, thank you. Yes, "notification" does seem a bit vague (as there can be other kinds of notifications), but "notifier" in this repo's context is pretty clear. Let me change that 👍